src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -803,4 +818,14 @@ public function hasAutowiringType($type)
 
         return isset($this->autowiringTypes[$type]);
     }
+
+    /**
+     * Returns all parts of this Definition that were explicitly configured.
+     *
+     * @return array An array of configured parts for this Definition
+     */
+    public function getConfiguredParts()
```

Comment:
should this replace getChanges on ChildDefinition? (or have getChanges migrate here instead?)